### ISSUE TITLE
Search bar overlays toolbar when right bar is active

### ISSUE THREAD

**Issue by pokaaaa** [yoyurec/logseq-solarized-extended-theme#1]
Many thanks for such a beautiful theme! It is definitely my favorite! :)

Two things that I would like to adjust:

1) Is there a way to reduce the length of the search bar? I can't access the buttons of the tool bar when the right bar is active, The search bar overlays toolbar (see screenshot attached).

2) How can I change the background photo if the theme is installed through the market place?

Sorry for my simple questions but I'm quite new with Logseq and I'm still trying to find my way in it.

Best regards,

pokaaa

![image](https://user-images.githubusercontent.com/104572473/167934753-91d5a01e-43b2-452e-96ea-d86c150cfbcf.png)

**Comment by yoyurec** [comment]
1. can you tell me LS & theme version?
2. readme updated - https://github.com/yoyurec/logseq-solarized-extended-theme#customizations

**Comment by pokaaaa** [comment]
thanks for your reply!I'm using LS 0.6.8 and theme 1.0.1.
Best,
pokaaa
    Em quarta-feira, 11 de maio de 2022 23:50:51 GMT+2, Yuriy Piskun ***@***.***> escreveu:  
 
 

   
   - can you tell me LS & theme version?
   - readme updated - https://github.com/yoyurec/logseq-solarized-extended-theme#customizations

**Comment by yoyurec** [comment]
try to update, current is 1.2.0

**Comment by yoyurec** [comment]
any news?

**Comment by pokaaaa** [comment]
I did update and the issues persist: I added the CSS code to my custom.css settings, changed the url and the background remains the same. The search bar still overlays with the tool bar. I have also noticed one additional issue: whenever I restart my Logseq, the system applies the solarized extended theme on top of my default theme. To solve this, I need to select again the solarized theme and apply the extension on top.Thanks a lot for your help! This theme looks amazing!

    Em sexta-feira, 13 de maio de 2022 02:19:08 GMT+2, Yuriy Piskun ***@***.***> escreveu:  
 
 


any news?

**Comment by yoyurec** [comment]
maybe try to delete theme and install again? also re-index graph... maybe its heavy cached

**Comment by yoyurec** [comment]
also check NEW bg code:
```
:root {
    --bg-url: url(..........);
}
```

**Comment by yoyurec** [comment]
can you post here your `custom.css`?

**Comment by pokaaaa** [comment]
done all the steps. Background now works! Length of the search bar has been reduced but still overlays with some icons on my toolbar. Perhaps you could only leave the search icon instead of the bar?I still have to change the theme when LS is restarted even after deleting and reinstalling the themes and re-indexing my graph.Thanks for your help!
    Em sexta-feira, 13 de maio de 2022 14:45:58 GMT+2, Yuriy Piskun ***@***.***> escreveu:  
 
 


can you post here your custom.css?

**Comment by yoyurec** [comment]
> Perhaps you could only leave the search icon instead of the bar
in narrow window size it reset to dafault sizes...
BUT you can configure it in css!
```
:root {
    --bg-url: url(......);
    --search-width: 500px; /* Search field size*/
    --search-field-text: 'Search or create page'; /* Maybe you wato to tranlate it ;) */
}
```

**Comment by yoyurec** [comment]
check latest version! no need to set width - it's responsive now.
uninstal/install theme on plugins page. not just update button. and restart Logseq

**Comment by pokaaaa** [comment]
Now it works!
The only problem that persists is when restarting LS. It returns to default theme every single time.
I've also found a minor bug: the the titles of the query tables when opened in the side bar don't wrap up the text. It's more aesthetics as this doesn't disturb my workflow. See screenshot attached.
Many thanks!



    Em sábado, 14 de maio de 2022 04:37:30 GMT+2, Yuriy Piskun ***@***.***> escreveu:  
 
 


Closed #1.

**Comment by yoyurec** [comment]
no screen ((
can you attach not by email, but on GitHub directly?

**Comment by pokaaaa** [comment]
Sure... sorry for this mistake... see screenshot attached... many thanks!

![image](https://user-images.githubusercontent.com/104572473/168549892-1055cf14-a7c9-4105-a36f-6d030f98ad66.png)

**Comment by yoyurec** [comment]
tnx. will fix

**Comment by yoyurec** [comment]
fixed in 1.6.7

**Comment by pokaaaa** [comment]
Looks amazing! MAny thanks!
    Em terça-feira, 17 de maio de 2022 00:50:51 GMT+2, Yuriy Piskun ***@***.***> escreveu:  
 
 


fixed in 1.6.7
